### PR TITLE
ADD Option for the IoT Agent tester to provision lists of devices

### PR DESCRIPTION
**Pull request by dmoranj**
Add Option for the IoT Agent tester to provision lists of devices

### CODE REVIEW

File: bin/iotAgentTester.js [hunk 1]
```diff
@@ -311,6 +312,95 @@ function provisionDevice(commands) {
...
+    }
+
+    function createPayload(csvData, callback) {
+        var dataList = parseDataFile(csvData),
+            payloadList = [];
+
+        async.map(dataList, function (item, innerCallback) {
+            var buffer;
+
+            mu
```
Comment (AlvaroVega) [inline bin/iotAgentTester.js]: poor!

**Comment by AlvaroVega** [comment]
LGTM

File: examples/ttClosedData.csv [hunk 2]
```diff
@@ -0,0 +1,5 @@
+DEVICE_ID DEVICE_NAME
```
Comment (XavierVal) [inline examples/ttClosedData.csv]: Should be public this info?

Comment (dmoranj) [inline examples/ttClosedData.csv]: Not really useful nor harmful. Don't think how can it be used for evil.

File: examples/ttClosedTemplate.json [hunk 3]
```diff
@@ -0,0 +1,82 @@
+{
+  "device_id": "{{deviceId}}",
+  "protocol": "TT_CLOSED",
+  "entity_name": "{{deviceName}}",
+  "entity_type": "{{deviceType}}",
+  "attributes": [
```
Comment (AlvaroVega) [inline examples/ttClosedTemplate.json]: Are the same attributes in TT_OPEN protocol? Sorry for off topic

Comment (dmoranj) [inline examples/ttClosedTemplate.json]: Not necessarily. TT_OPEN, as its name suggest, is open to include modules for different generic values. For TT_OPEN devices, the template must be tailored to the specific case. The same would be true for a "generic" TT Closed, i.e.: a TTClosed with a different set of modules as the ones we use. This template, anyway, has been uploaded as an example of how to create templates for the bulk loading, not to be a standard way of provisioning (see the "/examples" folder).

File: CHANGES_NEXT_RELEASE [hunk 4]
```diff
@@ -1 +1,2 @@
-- ADD Command attributes to the initial entity
\ No newline at end of file
+- ADD Command attributes to the initial entity
+- ADD Command option for the IoT Agent Tester to provision lists of devices.
```
Comment (gtorodelvalle) [inline CHANGES_NEXT_RELEASE]: Final blank line appreciated :)

**Comment by gtorodelvalle** [comment]
LGTM and merging :) Thanks @dmoranj !

**Pull request merged**